subprojects/docs/src/docs/userguide/api/kotlin_dsl.adoc

```diff
@@ -401,14 +401,14 @@ The <<plugins#sec:subprojects_plugins_dsl,Using Gradle plugins>> chapter explain
 [[ex:multi_project_ratpack]]
 .Declare plugin dependencies in the root build script using the `plugins {}` block
 ====
-include::sample[dir="snippets/kotlinDsl/multiProjectBuild/kotlin",files="settings.gradle.kts[tags=base];build.gradle.kts[tags=root];domain/build.gradle.kts[];infra/build.gradle.kts[];http/build.gradle.kts[]"]
+include::sample[dir="snippets/kotlinDsl/multiProjectBuildKotlin/kotlin",files="settings.gradle.kts[tags=base];build.gradle.kts[tags=root];domain/build.gradle.kts[];infra/build.gradle.kts[];http/build.gradle.kts[]"]
 ====
 
 If your build requires additional plugin repositories on top of the Gradle Plugin Portal, you should declare them in the `pluginManagement {}` block in your `settings.gradle.kts` file, like so:
 
 .Declare additional plugin repositories
 ====
-include::sample[dir="snippets/kotlinDsl/multiProjectBuild/kotlin",files="settings.gradle.kts[tags=repositories]"]
+include::sample[dir="snippets/kotlinDsl/multiProjectBuildKotlin/kotlin",files="settings.gradle.kts[tags=repositories]"]
```

Comment:
:x: instead of renaming, this ignore should be removed:

https://github.com/gradle/gradle/blob/7e76bf675007f6bd6a29ecd93ce7801ae522959d/subprojects/docs/docs.gradle#L595